docs/UserGuide.md

```diff
@@ -153,21 +153,21 @@ Examples:
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 * list followed by delete 1,1,2 deletes the 1st and 2nd person from current SocialBook.
 
-### Getting that parameter of all these people: `get` 
-Gets the specified parameter of the filtered people from SocialBook.
+### Getting the parameters of these people: `get` 
+Gets the specified parameters of each individual person in current person list.
 
 Format: `get PARAMETERS`
 
-* Gets the specified parameters for each person in the filtered list.
-* The only valid parameters are: n/, p/, e/, a/.
+* The only valid parameters are: name, phone, email, address.
+* Parameters are not case sensitive, which means `get PHONE name` will return phone number and name details too.
 * All inputted parameters must be valid for the details to be displayed. 
 * The details will be displayed in the order of the inputted parameters.
 * Executing the same parameter multiple times in the same command will still only return one set of that detail.
 
 Examples:
-* `list` followed by `get p/ e/` gets the phone numbers followed by the emails of the people in SocialBook.
-* `list` followed by `get p/ p/` gets only one set of the phone numbers of the people in SocialBook.
-* `find p/HIGH` followed by `get n/ a/` gets the names followed by the addresses of the high priority people.
+* `list` followed by `get phone EMAIL` gets the phone numbers followed by the emails of the people in SocialBook.
+* `list` followed by `get pHone phone` gets only one set of the phone numbers of the people in SocialBook.
```

Comment:
Is the first "pHone" substring meant to show the case-insensitive nature of the parameters?

Reply:
Yes, I believe so. The next example also uses mixed-case parameters.

Reply:
Yes, I believe so. The next example also uses mixed-case parameters.